pipeline/scripts/update-electron-checksum.js

```diff
@@ -34,6 +34,10 @@ const updateAllSha512s = async latestContent => {
 
     for (file of latestContent.files) {
         await updateSha512PropertyFromFile(file, file.url);
+        if (file.url.endsWith('zip')) {
```

Comment:
Instead of computing size and blockMapSize ourselves, we remove them. This has performance implications here: https://github.com/electron-userland/electron-builder/blob/master/packages/builder-util-runtime/src/updateInfo.ts (a few HTTP calls)

Reply:
see about 'size'